UI/Selenium/Hooks/Hooks.cs

```diff
@@ -315,24 +315,6 @@ public static void InsertReportingStepsApi(ScenarioContext scenarioContext)
             }
         }
 
-        [AfterScenario("Hearing")]
```

Comment:
Are any of these attributes with the value "hearing" required any more?

Reply:
This is not required, no hearing attributes anywhere in the feature files